### PR TITLE
Adds new theme [Neekster/MidnightTeal]

### PR DESCRIPTION
**Pull request by Neekster**
Adds Neekster/MidnightTeal theme

### CODE REVIEW

File: theme [hunk 1]
```diff
@@ -66,4 +66,5 @@
   "tgcowell/waves",
   "williamahartman/noctis-solarized",
   "wowgamr/animated-weather-card"
```
Comment (ZacheryThomas) [inline theme:68]: I you have to include a coma `,` at the end of this line. The Checks are failing since the json structure is a little off 👍

Comment (Neekster) [reply theme]: Thanks for letting me know what I had messed up! It's fixed now.

File: theme [hunk 2]
```diff
@@ -57,6 +57,7 @@
   "myleskeeffe/clear-theme-dark-vibrant",
   "naofireblade/clear-theme",
   "naofireblade/clear-theme-dark",
+  ""Neekster/MidnightTeal",
```
Comment (ZacheryThomas) [inline theme:60]: Looks like you've got an extra double quote `"` at the start of this repo name!

```suggestion
  "Neekster/MidnightTeal", 
```

**Comment by Neekster** [comment]
All checks passed 🎉

**Pull request merged**